app/views/annotation_categories/_annotation_category_edit.html.erb

```diff
@@ -19,6 +19,13 @@
         <%= f.label :annotation_category_name,
                     AnnotationCategory.human_attribute_name(:annotation_category_name) %>
         <%= f.text_field :annotation_category_name, rows: '3' %>
+        <%= f.label :flexible_criterion_id,
+                    t('annotation_categories.flexible_criterion_choice') %>
+        <%= f.collection_select :flexible_criterion_id,
+                                FlexibleCriterion.all.where(assignment: annotation_category.assignment),
```

Comment:
Generally prefer using associations when applicable, so `annotation_category.assignment.flexible_criteria`. Also use a `where` to filter by `ta_visible: true`.